_blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx

```diff
@@ -0,0 +1,54 @@
+---
+author: rescript-team
+date: "2021-06-25"
+title: Roadmap & New Landing Page
```

Comment:
it reads like
```
Roamdap & New
Landing Page
```
asymetric?

Reply:
should we go with `Roadmap 2021 & New Landing Page`?

Reply:
should be more symmetric now